AmplifyPlugins/Core/AWSPluginsCore/Model/Decorator/ModelDeleteDecorator.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+/// Decorate the GraphQLDocument with either a single id of `Model.Identifier`
+/// or with all fields part of a custom primary key for a "delete" mutation
+public struct ModelDeleteDecorator: ModelBasedGraphQLDocumentDecorator {
```

Comment:
as discussed you can just reuse `ModelIDDecorator`

Reply:
rename ideas, `ModelIdsDecorator` `ModelPrimaryIdDecorator` or keep the same? `ModelIdDecorator`